lib/command.c

```diff
@@ -1213,7 +1213,10 @@ static int handle_pipe_action(struct vty *vty, const char *cmd_in,
 			goto fail;
 		}
 		*cmd_out = XSTRDUP(MTYPE_TMP, cmd_in);
-		*(strstr(*cmd_out, "|")) = '\0';
+
+		char *pipe_loc = strstr(*cmd_out, "|");
+		if (pipe_loc)
+			*pipe_loc = 0;
```

Comment:
Coverity is giving a false positive here, see line 1188.

A better fix for the false positive, beyond marking it as such in Coverity, is:
```
char *u = cmd_out;
strsep(&u, "|");
```

Reply:
Is it ok to use POSIX string functions?

Reply:
Is it ok to use POSIX string functions?

Reply:
Yes

Reply:
Ok, I'm adding *cmd_out, because of the double pointer:
```

Reply:
Ok, I'm adding *cmd_out, because of the double pointer:
```
                char *u = *cmd_out;                                             
                strsep(&u, "|");                                                
```